notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Since we already searched for and saved the lightcurve files in the one above, could we save the filenames as an array (as in, during calc_rotation_rate) so we don't have to do it again here? Could probably speed up run time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>I think that's not true as long as you don't restart your kernel.</p>